build.gradle

```
@@ -1,5 +1,5 @@
 plugins {
-    id "eu.xenit.docker" version "5.1.1" apply false
+    id "eu.xenit.docker" version "5.2.0" apply false
```

```suggestion
    id "eu.xenit.docker" version "5.3.0" apply false
```

New release is out :smile:

---

whoops thought I was on the latest version of the gradle docker plugin.